app/_data/docs_nav_mesh_2.2.x.yml

```diff
@@ -133,35 +139,6 @@ items:
           - text: Performance fine-tuning
             url: /production/upgrades-tuning/fine-tuning/
             src: /.repos/kuma/app/_src/production/upgrades-tuning/fine-tuning/
-  - title: Install
-    icon: /assets/images/icons/documentation/icn-deployment-color.svg
-    items:
-      - text: Installation Options
-        url: /install/
-      - text: Kubernetes
-        url: /installation/kubernetes/
-      - text: Helm
-        url: /installation/helm/
-      - text: OpenShift
-        url: /installation/openshift/
-      - text: Docker
-        url: /installation/docker/
-      - text: Amazon ECS
-        url: /installation/ecs
-      - text: Amazon Linux
-        url: /installation/amazonlinux/
-      - text: Red Hat
-        url: /installation/redhat/
-      - text: CentOS
-        url: /installation/centos/
-      - text: Debian
-        url: /installation/debian/
-      - text: Ubuntu
-        url: /installation/ubuntu/
-      - text: macOS
-        url: /installation/macos/
-      - text: Windows
```

Comment:
@lahabana Do we need to keep Windows since that's Kong Mesh-specific?

Reply:
How was this supposed to fit in IA?

Reply:
@lahabana I think the "Install kumactl" and "Use Kuma" pages were meant to replace all the install pages, but once I got to the Mesh ones, I remembered that those pages weren't shared with the Kuma docs. They were separate. If there isn't a reason why they should be different/their own files, I think we can mirror what's currently in the Kuma nav and remove all the install topics and replace them with "Install kumactl" and "Use Kuma". 

Looking at the Windows one, I think we'd miss some of the Windows specifics in that file, but I think [Install kumactl](https://kuma.io/docs/2.2.x/production/install-kumactl/) does cover the general things like adding kumactl to your path.

Reply:
Ok I've added back a windows install page which seems to work. I've also did minor fixes to remove warnings